crates/key-value-azure/src/store.rs

```diff
@@ -446,8 +457,20 @@ impl AzureCosmosStore {
         query
     }
 
+    fn get_id_query(&self, key: &str) -> String {
+        let mut query = match self.store_id {
+            Some(_) => format!("SELECT c.id, c.store_id FROM c WHERE c.id='{key}'"),
+            None => format!("SELECT c.id FROM c WHERE c.id='{key}'"),
+        };
```

Comment:
Same as below. Can revert this to your original approach
```suggestion
        let mut query = format!("SELECT c.id, c.store_id FROM c WHERE c.id='{key}'");
```